.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Cache maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Build
+        run: mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent verify
```

Comment:
ik verwacht dat dit uiteindelijk mvn package wordt oid?
Dan kunnen we eventueel ook na iedere 'build' de resulterende jar als [github artifact](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) opslaan.

Reply:
Nee dit is bedoeld puur voor het verifieren van het buildbaar zijn van de code bij elke push naar een branch. Uiteindelijk zouden we op de main branch wel zoiets kunnen regelen.
(nice catch van de typefout ;))